example/swagger-files/parameter-styles-query.json

```diff
@@ -0,0 +1,247 @@
+                    }
+                }
+            }
+        },
+        "/explodedFormQueries": {
+            "get": {
+                "summary": "Demonstrate all exploded form query styles",
+                "operationId": "demonstrateQueries",
+                "parameters": [{
+                    "name": "primitive",
+                    "in": "query",
+                    "description": "Form style primitive",
+                    "required": true,
+                    "style": "form",
+                    "explode": true,
+                    "schema": {
+                        "type": "string"
+                    }
+                }, {
+                    "name": "array",
+                    "in": "query",
+                    "description": "Form style array",
+                    "required": true,
+                    "style": "form",
+                    "explode": true,
+                    "schema": {
+                        "type": "array",
+                        "items": {
+                            "type": "string"
+                        }
+                    }
+                }, {
+                    "name": "object",
+                    "in": "query",
+                    "description": "Form style object",
+                    "required": true,
+                    "style": "form",
+                    "explode": true,
+                    "schema": {
+                        "type": "object",
+                        "properties": {
+                            "name": {
+                                "type": "string"
+                            },
+                            "description": {
+                                "type": "string"
+                            }
+                        }
+                    }
+                }],
+                "responses": {
+                    "200": {
+                        "description": "Successful response"
+                    }
+                }
+            }
+        },
+        "/nonExplodedSpaceDelimitedQueries": {
+            "get": {
+                "summary": "Demonstrate all non-exploded spaceDelimited query styles",
+                "operationId": "demonstrateQueries",
+                "parameters": [{
+                    "name": "array",
+                    "in": "query",
+                    "description": "spaceDelimited style array",
+                    "required": true,
+                    "style": "spaceDelimited",
+                    "explode": false,
+                    "schema": {
+                        "type": "array",
+                        "items": {
+                            "type": "string"
+                        }
+                    }
+                }, {
+                    "name": "object",
+                    "in": "query",
+                    "description": "spaceDelimited style object. Note: This does not currently work. See associated test.",
```

Comment:
Haha I was like "hmm this isn't working" and then noticed this comment.

![Screen Shot 2021-01-06 at 10 42 56 AM](https://user-images.githubusercontent.com/33762/103807932-0645cf00-500c-11eb-8b49-8158f8157080.png)

Reply:
Yeah that's one of the ones called out in the additional notes. Seems to be a problem with the encoding library we use. I can easily fix it, but hesitated.